SharedPackages/BrowserServicesKit/Sources/PixelKit/WideEvent/WideEventParameters.swift

```diff
@@ -89,24 +89,14 @@ public enum WideEventParameter {
     }
 
     public enum Feature {
-        static let name = "feature.name"
-        static let status = "feature.status"
-        static let statusReason = "feature.status_reason"
-
-        static let errorDomain = "feature.data.error.domain"
-        static let errorCode = "feature.data.error.code"
-        static let underlyingErrorDomain = "feature.data.error.underlying_domain"
-        static let underlyingErrorCode = "feature.data.error.underlying_code"
-    }
-
-    public enum SubscriptionFeature {
```

Comment:
Subscription params are moved into the subscription library, along with its wide event.